lib/galaxy/datatypes/media.py

```diff
@@ -32,6 +33,94 @@ def ffprobe(path):
     return data["format"], data["streams"]
 
 
+magic_number = {
+    "mp4": {
+        "offset": 4,
+        "hex": [
+            "66 74 79 70 69",
+            "66 74 79 70 6D",
+            "66 74 79 70 4D",
+        ],
+    },
+    "flv": {"offset": 0, "hex": ["46 4C 56 01"]},
```

Comment:
also by [wikipedia's list](https://en.wikipedia.org/wiki/List_of_file_signatures), the magic should just be `FLV`, so `01` in the last byte might be too restrictive.

Since many of these are strings it might make sense for your implementation to support something like

```suggestion
    "flv": {"offset": 0, "string": ["FLV"]},
```